mobile/src/components/home/DiscoverBar.jsx

```diff
@@ -0,0 +1,84 @@
+import React from 'react';
+import {View, Text, StyleSheet, Image, TouchableOpacity, Linking} from 'react-native';
+import {colors, globalStyleSheet} from '../../utilites/Theme';
+
+const DiscoverBar = () => {
+  return (
+    <View style={styles.DiscoverRow}>
+
+      <TouchableOpacity style={styles.container} onPress={() => Linking.openURL('https://isa.ualberta.ca/')}>
+        <Image
+          source={require('../../../assets/Website.png')}
```

Comment:
the image is a bit blurry, could we try using .svg instead of .png?

Reply:
I haven't done this, was having troubles installing some packages, I can fix this in my other ticket (consolidate home page) if that works better. I'll probably try to make it less blurry using vector icons.

Reply:
sounds good!